utils/aur/check-aur-dependencies.py

```diff
@@ -34,7 +35,7 @@ async def check_pkg(session: aiohttp.ClientSession, num: int, pkg: str) -> Tuple
 
 
 async def main() -> int:
-    parser = argparse.ArgumentParser(description='check-aur.py')
+    parser = argparse.ArgumentParser(description='check-aur-dependencies.py')
```

Comment:
```suggestion
    parser = argparse.ArgumentParser(description=sys.argv[1])
```
Is much better?

Reply:
You can refer to [this link](https://docs.python.org/zh-cn/3/library/argparse.html#description). Just modify it to adapt to your change. Unless you rename again and...